topics/qt/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: qt-framework, cute, qt-application
+created_by: Haavard Nord, Eirik Chambe-Eng
+display_name: Qt
+github_url: https://github.com/qt
+logo: Qt_logo.png
+released: 20 May 1995
+short_description: Qt is a cross-platform application framework that is used for developing application software.
```

Comment:
This could be rephrased so it isn't the same as the Wikipedia page. What about:

> Qt, pronounced "cute", is a cross-platform application development framework.